config/boards/coolpi-cm5.csc

```diff
@@ -0,0 +1,40 @@
+# Rockchip RK3588 SoC octa core 4-16GB SoC 2.5GBe PoE eMMC USB3 NVME
+BOARD_NAME="CoolPi CM5"
+BOARDFAMILY="rockchip-rk3588"
+BOARD_MAINTAINER="Andy Yan"
+BOARD_FIRMWARE_INSTALL="-full"
+BOOT_SOC="rk3588"
+BOOTCONFIG="coolpi-cm5-evb-rk3588_defconfig"
+KERNEL_TARGET="edge"
+FULL_DESKTOP="yes"
+BOOT_LOGO="desktop"
+BOOT_FDT_FILE="rockchip/rk3588-coolpi-cm5-evb.dtb"
+BOOT_SCENARIO="spl-blobs"
+BOOT_SUPPORT_SPI="yes"
+BOOT_SPI_RKSPI_LOADER="yes"
+IMAGE_PARTITION_TABLE="gpt"
+BOOTFS_TYPE="ext4"
```

Comment:
This will create an ext4 boot partition. But I think a single root partition should be okay with mainline uboot.

Reply:
Hi Jianfeng，
Thanks for your review。
Does that mean the downstream u-boot does not support a single root partition？
I would like to keep it as it is if an ext4 boot partition has no other limitation。

Reply:
An old version of rk3568 vendor u-boot does not support ext4 partition well, so we defined `BOOTFS_TYPE="fat"` for some rk3568 boards like rock-3a. But rk3588's vendor u-boot doesn't have this issue.
Single boot partition will have issue when multi kernel are installed because of its size limitation.

Reply:
So should I just remove BOOTFS_TYPE here ？

Reply:
If there is no issue, you can remove it.

Reply:
I removed it, and the system seems boot fine.
So i remove it in this version.